spec/models/active_shipping_calculator_spec.rb

```diff
@@ -126,31 +124,35 @@ module ActiveShipping
         calculator.compute(package)
       end
 
-      xit "should check the cache first before finding rates" do
-        Rails.cache.fetch(calculator.send(:cache_key, order)) { Hash.new }
+      it "should check the cache first before finding rates" do
+        Rails.cache.fetch(calculator.send(:cache_key, package)) { Hash.new }
         carrier.should_not_receive(:find_rates)
         calculator.compute(package)
       end
 
       context "with valid response" do
+        let(:rates) do
+          [ double('rate', :service_name => "Bogus Calculator", :price => 999) ]
+        end
+
         before do
           carrier.should_receive(:find_rates).and_return(response)
```

Comment:
Would prefer if this used the new rspec syntaxt `allow(carrier).to receive(:find_rates)`.